test/e2e/space_cleanup_test.go

```diff
@@ -37,11 +38,15 @@ func TestSpaceAndSpaceBindingCleanup(t *testing.T) {
 
 		t.Run("when mur is deleted", func(t *testing.T) {
 			// given
-			_, mur, spaceBinding := setupForSpaceBindingCleanupTest(t, awaitilities, memberAwait, "lara", "ibm")
+			_, userSignup, spaceBinding := setupForSpaceBindingCleanupTest(t, awaitilities, memberAwait, "lara", "ibm")
 
 			// when
-			err := hostAwait.Client.Delete(context.TODO(), mur)
```

Comment:
that was a bad mistake :man_facepalming: 
no idea how I could think that it could work :smile: :man_shrugging: 
Thanks for fixing it :+1: